index.html

```diff
@@ -33,6 +33,105 @@ <h1 class="header2">I’m Iptisam
+
+            <div class="container2">
+                <div class="rect2"></div>
+                <h3 class="Proj2">Multi-Post Stories</h3>
+                <ul class="list2">
+                    <li class="frame1"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang1">
+                    <li class="HTML1"><a href="#">html</a></li>
+                    <li class="css1"><a href="#">CSS</a></li>
+                    <li class="Java1"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box1">
+                <li class="button1"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+
+            <div class="container3">
+                <div class="rect3"></div>
+                <h4 class="Proj3">Tonic</h4>
+                <ul class="list3">
+                    <li class="frame2"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang2">
+                    <li class="HTML2"><a href="#">html</a></li>
+                    <li class="css2"><a href="#">CSS</a></li>
+                    <li class="Java2"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box2">
+                <li class="button2"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+
+            <div class="container4">
+                <div class="rect4"></div>
+                <h5 class="Proj4">Multi-Post Stories</h5>
+                <ul class="list4">
+                    <li class="frame3"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang3">
+                    <li class="HTML3"><a href="#">html</a></li>
+                    <li class="css3"><a href="#">CSS</a></li>
+                    <li class="Java3"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box3">
+                <li class="button3"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="container5">
+                <h6 class="header3">About Myself</h6>
+                <p class="descrip">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="links1">LET’S CONNECT</p>
+                <ul class="images1">
+                    <li class="shape12"><a href="#"><img src="Images/Frame1.png" alt="shape"></a></li>    
```

Comment:
- [ ] Kindly list all your icons in this section you are not supposed to use an image to replace them, each of them should be clickable.The icons use a ul tag to group them, and li per icon; each icon has a link (a tag)